worker-document-testing-unit/src/test/java/com/hpe/caf/worker/document/testing/TestDocumentWorker.java

```diff
@@ -85,5 +91,12 @@ public void processDocument(Document document) throws InterruptedException, Docu
         for (FieldValue fieldValue : fieldValues) {
             field.add(fieldValue.getStringValue());
         }
+        if (document.hasSubdocuments()) {
```

Comment:
The `if` check seems a bit irrelevant since `.size()` will come back as 0.

Reply:
Yep, I looked into the implementation.